scripts/python/createcfg.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python
+
+import os 
+import json
+import argparse 
+
```

Comment:
Good write comment saying:

This script aims to generate the config json file which has all these fields.